From 002136b0acaff18431b7f29c147b8358edf870a8 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Tue, 24 Sep 2013 13:16:24 -0700
Subject: ueventd: read ueventd.composite.rc

This file is created at build time by the mixins mechanism.
Since ueventd config syntax doesn't support import statements,
add code to read the file here immediately after the base
ueventd.rc file.

Issue: ADEV-21
Change-Id: I6f354a03e38d130689cc4d716eb20c8c3f7e3140
Category: AOSP Improvement
Domain: mixins, AOSP-ueventd
Upstream-candidate: Yes
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 init/ueventd.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/init/ueventd.c b/init/ueventd.c
index 6f2574f..9da1e8e 100644
--- a/init/ueventd.c
+++ b/init/ueventd.c
@@ -124,6 +124,7 @@ int ueventd_main(int argc, char **argv)
     get_hardware_name(hardware, &revision);
 
     ueventd_parse_config_file("/ueventd.rc");
+    ueventd_parse_config_file("/ueventd.composite.rc");
 
     snprintf(tmp, sizeof(tmp), "/ueventd.%s.rc", hardware);
     ueventd_parse_config_file(tmp);
-- 
1.7.4.1

